frontend/src/main/webapp/static/js/service/poll_service.js

```diff
@@ -19,6 +19,8 @@ angular.module('myApp').factory('PollService', ['$http', '$q', function($http, $
             .then(
```

Comment:
@line5     "var REST_SERVICE_URI = 'http://localhost:8080/poll/';"  is wrong in this context. 

1. don't hardcode the protocol or host in your static browser code resources. This should just be "poll/", as a relative path and the server should do  the rest. 
2. you should not be directly connecting to the provider (8080), you should still be connecting to the controllers in your front-end, and brokering that data from the RestTemplate located in the frontend Manager classes. I suspect this is the root problem that triggered adding @crossorgin annotations to your controller endpoints.